cli/cmd/top.go

```diff
@@ -483,20 +484,29 @@ func processEvents(eventCh <-chan pb.TapEvent, requestCh chan<- topRequest, done
 	}
 }
 
-func pollInput(closing chan<- struct{}) {
+func pollInput(done chan<- struct{}, horizontalScroll chan int) {
 	for {
 		switch ev := termbox.PollEvent(); ev.Type {
 		case termbox.EventKey:
 			if ev.Ch == 'q' || ev.Key == termbox.KeyCtrlC {
-				closing <- struct{}{}
+				close(done)
 				return
 			}
+			if ev.Ch == 'a' || ev.Key == termbox.KeyArrowRight {
```

Comment:
Looks like the `a` key and right don't match here, as they are opposite in placement

Reply:
I just tested and pressing 'a' or Right arrow scrolls the view port to the left.  I think it should be that 'a' or Right should move the view port to the right (i.e. move the text to the left)

Reply:
I have corrected this now.